hermes_core/util/tests/test_schema.py

```diff
@@ -306,24 +306,24 @@ def test_min_max_TT2000():
     """Get min/max values for TT2000 types"""
     minval, maxval = HermesDataSchema()._get_minmax(const.CDF_TIME_TT2000)
     # Make sure the minimum isn't just plain invalid
-    assert minval == datetime.datetime(1, 1, 1)
-    assert maxval == datetime.datetime(2292, 4, 11, 11, 46, 7, 670776)
+    assert minval == datetime.datetime(1900, 1, 1, 0, 0, 0, 0)
+    assert maxval == datetime.datetime(2250, 1, 1, 0, 0, 0, 0)
 
 
 def test_min_max_Epoch16():
     """Get min/max values for Epoch16 types"""
     minval, maxval = HermesDataSchema()._get_minmax(const.CDF_EPOCH16)
     # Make sure the minimum isn't just plain invalid
-    assert minval == datetime.datetime(1, 1, 1)
-    assert maxval == datetime.datetime(9999, 12, 31, 23, 59, 59)
+    assert minval == datetime.datetime(1900, 1, 1, 0, 0, 0, 0)
```

Comment:
Isn't this 01-Jan-0000 00:00:00.000.000.000.000?

Reply:
See page 57 of https://spdf.gsfc.nasa.gov/pub/software/cdf/doc/cdf390/cdf390ug.pdf

Reply:
The same is true for epoch 8 bit.

Reply:
The table on Page 57 looks like its referring to the FILLVAL attribute rather than the VALIDMIN/MAX attributes. From a brief skim through the PDF it does not look like there are recommendations for VALIDMIN/MAX default values. 

I updated these here because we were getting overflow errors when loading and saving CDF files so I updated them to more reasonable MIN and MAX values.

Reply:
Sorry i misunderstood the spec.